cli/cli.go

```diff
@@ -308,11 +323,17 @@ func printAPIKeyNotSetMessage() {
 	styleRed := lipgloss.NewStyle().Foreground(lipgloss.Color("9"))
 
 	msg1 := styleRed.Render("OPENAI_API_KEY environment variable not set.")
-	msg2, _ := r.Render(`
+
+	// make it platform agnostic
+	message_string := fmt.Sprintf(`
 	1. Generate your API key at https://platform.openai.com/account/api-keys
 	2. Add your credit card in the API (for the free trial)
 	3. Set your key by running:
-	` + "\n```bash\nexport OPENAI_API_KEY=[your key]\n```\n" + "    4. (Recommended) Add that ^ line to your .zshrc or .bashrc file.\n")
+	%s
+	4. (Recommended) Add that ^ line to your %s file(s).
+	5. If you are a member of an organization do the same thing but with OPENAI_ORGANIZATION_KEY`, getShellSyntax(), getProfileScriptName())
```

Comment:
I expect most users to be individuals, so to keep things clean I'd rather not add 5 here, but instead add it potentially to an error message a user would get if they don't set the org key. If you can figure out what http error that is then you could also add that yourself! Or lmk and I can as well.

Reply:
We could easily check for it. I'll do some digging.